### ISSUE TITLE
Imported NPM repositories don't maintain state for retrieval_cache_period_seconds

### ISSUE THREAD

**Issue by nrowealign** [jfrog/terraform-provider-artifactory#605]
**Describe the bug**
Imported NPM repositories return retrieval_cache_period_seconds to a default value.

**Requirements for and issue**
Similar to #593 imported NPM repositories lose state for retrieval_cache_period_seconds; setting it to a default value even if that is the current value.

Here is the output of `terraform show` for the given resource:
```
# artifactory_virtual_npm_repository.npm-thirdparty:
resource "artifactory_virtual_npm_repository" "npm-thirdparty" {
    artifactory_requests_can_retrieve_remote_artifacts = true
    excludes_pattern                                   = "**/[trimmed]"
    external_dependencies_enabled                      = false
    external_dependencies_patterns                     = [
        "**",
    ]
    id                                                 = "npm-thirdparty"
    includes_pattern                                   = "**/*"
    key                                                = "npm-thirdparty"
    package_type                                       = "npm"
    project_environments                               = []
    repo_layout_ref                                    = "npm-default"
    repositories                                       = [
        "npm-proxy-npmjs",
        "npm-proxy-npmjs-approved",
    ]
}
```

Here is the output of `terraform plan` for the same resource:
```
  # artifactory_virtual_npm_repository.npm-thirdparty will be updated in-place
  ~ resource "artifactory_virtual_npm_repository" "npm-thirdparty" {
        id                                                 = "npm-thirdparty"
      + retrieval_cache_period_seconds                     = 7200
        # (10 unchanged attributes hidden)
    }
```

As before I corrected the value for retrieval_cache_period_seconds in the config, but the `plan` still thinks a change is necessary.

Artifactory verssion 7.46.10
Terraform version 1.2.5
Provider version 6.21.6

**Expected behavior**
The state should reflect the current config correctly.

**Additional context**

**Comment by danielmkn** [comment]
It was fixed in v6.24.2

**Comment by nrowealign** [comment]
This same problem is now happening with remote_repo_layout_ref. I know you're going to ask me to file another ticket, but I probably won't have time for the next couple of weeks.

**Comment by danielmkn** [comment]
@nrowealign, it was fixed in 6.23.0
